### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by industrian**
This PR cleans up the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-This repository contains scripts that apply boilerplate `key` values to resources.
-
 # What is this useful for?
 
 Import and export functionality in Composable Commerce uses the `key` field as the identifier for resources. If your resources do not have a `key`, they cannot be updated via import and their references may be broken in exported files.
 
 As the `key` value is optional, resources within your Composable Commerce Project may lack them.
 
-These scripts apply boilerplate `key` values to resources, which enables you to fully use the import and export functions within Composable Commerce.
+This app sets boilerplate `key` values to resources, which enables you to fully use the import and export functions within Composable Commerce.
```
Comment (stmeissner) [inline README.md:7]: Are these random values, or do they follow some kind of a pattern?
It is explained in the the very last line of the part to be removed:
>This script applies `key` values using the format: `{resourceType}_{resourceId}`.

Is it no longer valid?

Comment (industrian) [reply README.md:7]:
```suggestion
This app sets boilerplate `key` values to resources, which enables you to fully use the import and export functions within Composable Commerce. The boilerplate `key` values use the format: {resourceType}_{resourceId}.
```

Comment (industrian) [reply README.md:7]: I re-added the format to an info box.

**Review by stmeissner — APPROVED** [review]
LGTM 👍🏻

**Review by agonbina — APPROVED** [review]

**Pull request merged**